### PR TITLE
Bug 1284174 - [TV][Home] Disable "decks can be moved into folder" feature

### PR DESCRIPTION
**Pull request by luke-chang**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=d5a4844dd39f42c3a99b1d28bfe1b24d0d8fbb11

File: tv_apps/smart-home/js/home.js [hunk 1]
```diff
@@ -815,8 +817,14 @@
     },
 
     checkAddFolderButton: function() {
-      this.addFolderButton.classList.toggle('hidden',
-        !!this.mode || !this.cardManager.hasCardInCardList());
+      if (this._checkAddFolderButtonTimer) {
+        clearTimeout(this._checkAddFolderButtonTimer);
+      }
+      this._checkAddFolderButtonTimer = setTimeout(() => {
```
Comment (rexboy7) [inline tv_apps/smart-home/js/home.js]: Please add comments to describe what setTimeout is trying to resolve.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=a133bbe1b21bca130b768990d2c4c44ba45815a4

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=a133bbe1b21bca130b768990d2c4c44ba45815a4

**Pull request merged**